drivers/hmis/lib/form_data/records/inventory.json

```diff
@@ -1,6 +1,26 @@
 {
   "name": "inventory",
   "item": [
+    {
+      "type": "DATE",
+      "link_id": "project-start",
+      "hidden": true,
+      "initial": [
+        {
+          "value_local_constant": "$projectStartDate"
+        }
+      ]
+    },
+    {
+      "type": "DATE",
+      "link_id": "project-end",
+      "hidden": true,
+      "initial": [
+        {
+          "value_local_constant": "$projectEndDate"
```

Comment:
It seems unlikely you'd be adding an inventory record to a closed project, but I guess there's no harm in setting the end date to match if you did.  (Or maybe I'm reading this all wrong, and this is about validation, not initial data)